go.mod

```diff
@@ -74,18 +74,18 @@ require (
 	go.uber.org/atomic v1.7.0 // indirect
 	go.uber.org/multierr v1.6.0 // indirect
 	go.uber.org/zap v1.24.0 // indirect
-	golang.org/x/net v0.8.0 // indirect
-	golang.org/x/oauth2 v0.3.0 // indirect
+	golang.org/x/net v0.12.0 // indirect
+	golang.org/x/oauth2 v0.7.0 // indirect
 	golang.org/x/sync v0.1.0 // indirect
-	golang.org/x/sys v0.6.0 // indirect
-	golang.org/x/term v0.6.0 // indirect
-	golang.org/x/text v0.8.0 // indirect
+	golang.org/x/sys v0.10.0 // indirect
+	golang.org/x/term v0.10.0 // indirect
+	golang.org/x/text v0.11.0 // indirect
 	golang.org/x/time v0.3.0 // indirect
 	gomodules.xyz/jsonpatch/v2 v2.2.0 // indirect
 	google.golang.org/appengine v1.6.7 // indirect
-	google.golang.org/genproto v0.0.0-20220502173005-c8bf987b8c21 // indirect
-	google.golang.org/grpc v1.49.0 // indirect
-	google.golang.org/protobuf v1.28.1 // indirect
+	google.golang.org/genproto/googleapis/rpc v0.0.0-20230717213848-3f92550aa753 // indirect
+	google.golang.org/grpc v1.56.2 // indirect
```

Comment:
I don't think we should be updating these indirect dependencies since there may be unforeseen side effects.  We should be updating the main dependencies that pick up the latest grpc module.  If you run a `go mod graph | grep google.golang.org/grpc` on the module, you'll see that it's the registry index generator and registry library that are pulling in grpc.   If you run the same command on the registry generator and registry library modules, you can determine that the main dependency `k8s.io/apiextensions-apiserver v0.26.1` is the problem.  We need to update this module but it doesn't seem like a patch update is available yet: https://github.com/kubernetes/apiextensions-apiserver/blob/release-1.27/go.mod#L23

Reply:
Good catch, I'll put a hold on all of these PRs until a patch on the direct dependency is available. Also, will change the status of the issue to blocked.